src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/FileBasedPrograms/VirtualProjectXmlProvider.cs

```diff
@@ -203,13 +203,13 @@ internal static string MakeVirtualProjectContent_DirectFallback(string documentF
         Contract.ThrowIfFalse(PathUtilities.IsAbsolute(documentFilePath));
         var artifactsPath = GetArtifactsPath(documentFilePath);
 
-        var targetFramework = Environment.GetEnvironmentVariable("DOTNET_RUN_FILE_TFM") ?? "net10.0";
+        var targetFramework = Environment.GetEnvironmentVariable("DOTNET_RUN_FILE_TFM") ?? "net$(BundledNETCoreAppTargetFrameworkVersion)";
```

Comment:
`BundledNETCoreAppTargetFrameworkVersion` is part of each SDK. For .NET 10 it is `10.0`